### PR TITLE
yet more pch

### PR DESCRIPTION
**Pull request by mck1117**
this change alone knocks ~30% off the firmware compile time on my machine (with a single threaded build, to try and reduce the impact of linking). 1m54 before -> 1m18 after.

depends on #3067

### CODE REVIEW

**Comment by rusefillc** [comment]
what about #3069

**Comment by rusefillc** [comment]
@mck1117 please rebase
@mck1117 please help this PR

**Comment by mck1117** [comment]
ooh it's green!

File: firmware/console/binary/signature.cpp [hunk 1]
```diff
@@ -29,7 +29,7 @@
 #define SIGNATURE_H QUOTE(SIGNATURE_NAME SHORT_BOARD_NAME SIGNATURE_EXT)
 
 #if !EFI_UNIT_TEST
-#include SIGNATURE_H
+#include "signature_proteus_f4.h"
```
Comment (rusefillc) [inline firmware/console/binary/signature.cpp:32]: @Abricosvw say "huh?" #3115

**Pull request merged**